app/src/main/res/layout/fragment_mentorship_tasks.xml

```diff
@@ -114,7 +114,10 @@
         android:layout_height="wrap_content"
         android:layout_alignParentEnd="true"
         android:layout_alignParentBottom="true"
-        android:layout_margin="10dp"
+        android:layout_marginTop="10dp"
```

Comment:
@jddeep settting margins is probably not the best solution here as it may fail to work as expected for few devices. Instead you can modify the [`activity_main.xml`](https://github.com/anitab-org/mentorship-android/blob/develop/app/src/main/res/layout/activity_main.xml) file to make use of `LinearLayout` for placing the FrameLayout and BottomNavigation bar vertically